stacker/lookups/handlers/dynamodb.py

```diff
@@ -74,35 +56,100 @@ def handler(value, **kwargs):
         )
     except ClientError as e:
         if e.response['Error']['Code'] == 'ResourceNotFoundException':
-            raise ValueError('Cannot find the dynamodb table: ' + table_name)
+            raise ValueError(
+                'Cannot find the dynamodb table: {}'.format(table_name))
         elif e.response['Error']['Code'] == 'ValidationException':
-            raise ValueError('No dynamo record matched that partition key')
-
+            raise ValueError(
+                'No dynamodb record matched the partition key: '
+                '{}'.format(table_lookup))
+        else:
+            raise ValueError('The dynamodb lookup {} had an error: '
+                             '{}'.format(value, e))
     # find and return the key from the dynamo data returned
     if 'Item' in response:
-        if len(response['Item']) > 0:
-            return (_getValFromDict(response['Item'], new_keys[1:]))
-        else:
-            raise ValueError('The specified dynamo record could not be found')
+        return (_get_val_from_ddb_data(response['Item'], new_keys[1:]))
     else:
         raise ValueError(
-            'No dynamo record with those paramters could be found')
+            'The dynamodb record could not be found using the following '
+            'key: {}'.format(new_keys[0]))
+
+
+def _lookup_key_parse(table_keys):
+    """Return the order in which the stacks should be executed.
+
+    Args:
+        - dependencies (dict): a dictionary where each key should be the
```

Comment:
The `-` at the beginning here isn't part of the standard, can you remove it?  Thanks.